client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -86,6 +113,18 @@ class StatsPostDetail extends Component {
 				/>
 
 				<PostWeeks siteId={ this.props.siteId } postId={ this.props.postId } />
+				<WebPreview
```

Comment:
I think all websites are not previewable, (Jetpack non https and also iframes can be explicitely blocked). We had issues with that when working on taxonomies previews. I think now there's a Component that take care of these issues `WithPreviewProps`. More background in #10205